application/controllers/CertificatesController.php

```diff
@@ -68,7 +68,7 @@ public function indexAction()
             $filter = $searchBar->getFilter();
         }
 
-        $certificates->peekAhead($this->view->compact);
+        $certificates->peekAhead($this->view->compact ?? false);
```

Comment:
compact is always a bool (`ActionController::__construct()`). If phpstan still throws an error, this should be added to the baeline.